pages/api/profiles.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from '@prisma/client'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+const prisma = new PrismaClient()
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  if (req.method !== 'GET') {
+    return res.status(405).json({ message: 'Method Not Allowed' })
+  }
+
+  try {
+    const publishedRecords = await prisma.profile.findMany({
```

Comment:
**nitpick:** returned value won't === `ProfilePayload`. You need to serialize returned value, and also include `Profile` relations